source/core/replica-set-high-availability.txt

```diff
@@ -15,29 +19,242 @@ Replica Set High Availability
    :depth: 1
    :class: singlecol
 
-:term:`Replica sets <replica set>` provide high availability using
-automatic :term:`failover`. Failover allows a :term:`secondary` member
-to become :term:`primary` if the current primary becomes unavailable.
+:term:`Replica sets <replica set>` use elections to select the 
+:term:`primary` for the cluster. Replica set elections support
+high availability by allowing clusters to automatically recover 
+normal operations after a primary failover event. Applications can
+continue to interact with the cluster during elections, mitigating
+the risk of downtime due to critical errors such as network 
+disruptions and hardware failure.
+
+Replica sets can trigger an election in response to a variety of events,
+such as:
+
+- :method:`initiating a replica set <rs.initiate()>`,
+- performing replica set maintenance using methods such as :method:`rs.stepDown()` or :method:`rs.reconfig()`, and
+- losing connectivity to the primary for more than the configured :rsconf:`timeout <settings.electionTimeoutMillis>` (10 seconds by default).
+
+In the following diagram, the primary node unavailability
+exceeded the 
+:rsconf:`configured timeout <settings.electionTimeoutMillis>`.
+One of the remaining secondaries holds an election in response to
```

Comment:
s/holds/calls for.  everyone's involved in the election, just one node initiated it

Reply:
Fixed. also made a tweak, as the sentence as is implies the secondary that calls for an election is the one that would win. Unless its technically accurate to say that.

Reply:
When a node calls for an election, it is fundamentally nominating itself.  But if multiple nodes call for an election at the same time, there's no guarantee which (if any) of them wins.  I think the way you reworded it is fine

Reply:
Fixed, thanks.